src/scss/_settings.scss

```diff
@@ -13,4 +13,4 @@ $increase-font-size-on-larger-screens: false;
 $side-panel-z-index: 101;
 $side-navigation-z-index: 103;
 $application-layout--breakpoint-side-nav-expanded: $breakpoint-xx-large;
-$assets-path: "~/assets/fonts/";
+$assets-path: "https://assets.ubuntu.com/v1/assets/fonts/";
```

Comment:
We cannot use remote assets as maas-ui is meant to work without internet connection.